pkg/tsdb/elasticsearch/response_parser.go

```diff
@@ -157,6 +158,20 @@ func processLogsResponse(res *es.SearchResponse, target *Query, configuredFields
 	sortedPropNames := sortPropNames(propNames, configuredFields, true)
 	fields := processDocsToDataFrameFields(docs, sortedPropNames, configuredFields)
 
+	// copy `_id` field to `id` field
+	idIdx := slices.IndexFunc(fields, func(f *data.Field) bool {
+		return f.Name == "_id"
+	})
+	if idIdx != -1 {
+		idField := fields[idIdx]
+		fieldVector := make([]*string, idField.Len())
+		for i := 0; i < idField.Len(); i++ {
```

Comment:
Is this copying a string char by char? Because it's not a string but a `[]*string`? Why?

Reply:
Not characters but strings, so it makes more sense.